lib/time/time.go

```diff
@@ -10,16 +15,54 @@ import (
 	"go.starlark.net/syntax"
 )
 
-// Module time is a Starlark module of time-related functions.
+// Module time is a Starlark module of time-related functions and constants.
+// The module defines the following functions:
+//
+//     from_timestamp(sec, nsec) - Converts the given Unix time corresponding to the total amount of seconds
+//                                 and nonoseconds since January 1, 1970 UTC into an object of type Time.
+//                                 The total amount of nonoseconds is optional.
+//                                 It is valid to pass nonoseconds outside the range [0, 999999999].
+//                                 Not all sec values have a corresponding time value. One such
+//                                 value is 1<<63-1 (the largest int64 value).
+//
+//     is_valid_timezone(loc) - Indicates whether the given name of location is valid or not.
+//                              Returns true if it is valid, false otherwise.
+//
+//     now() - Returns the current local time.
+//
+//     parse_duration(d) - Parses the given duration string.
+//                         A duration string is a possibly signed sequence of decimal numbers,
+//                         each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m".
+//                         Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+//
+//     parseTime(x, format, location) - Parses the given time string using a specific time format and location.
```

Comment:
parse_time

The design of this function is quite Go-specific, and I wonder whether it was a good idea to expose it directly. In any case, it needs more documentation about how the format string is interpreted. For now, let's redirect readers to Go's time.Parse function, since that is the actual behavior, though this is not ideal for a Starlark spec.